tests/lido/ethereum/lido_ethereum_accounting_test.sql

```diff
@@ -0,0 +1,17 @@
+with  tests as (
+select  'Check Lido deposits' as test_name, 
+case when sum(amount_staked) = 224000000000000000000 then true else false end as success
```

Comment:
And the result of `sum()` may change as well, and this test may not be working properly.